training/index.html

```diff
@@ -0,0 +1,273 @@
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="{% link discover/resources/index.md %}">Discover Server Resources and Information</a>: Once users have signed up for Discover, they can check out this page to learn everything there is to know about consuming the services available from Discover.</li>
+    </ul>
+  </section>
+  <section class="training-next">
+    <h2 class="text-2xl caps">Training</h2>
+  </section>
+  <section class="training-left">
+    <h3 class="text-xl"><a href="https://esri.com/training" rel="noopener noreferrer" target="_blank">Online Esri Training</a></h3>
+    <p> Esri has many online courses that are included in the annual software maintenance cost. They also have several basic free courses that are a good introduction to GIS and ArcMap/ ArcGIS Pro.</p>
+    <h3 class="text-xl"><a href="https://www.esri.com/training/" rel="noopener noreferrer" target="_blank">Free Esri Courses</a></h3>
+    <ul class="fa-ul">
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span>Intro to GIS</li>
+      <ul class="fa-ul">
+        <li><span class="fa-li text-blue"><i class="fas fa-compass"></i></span><a href="https://www.esri.com/training/catalog/5d9cd7de5edc347a71611ccc/gis-basics/" rel="noopener noreferrer" target="_blank">GIS Basics</a> (ArcGIS Pro)</li>
+        <li><span class="fa-li text-blue"><i class="fas fa-compass"></i></span><a href="https://www.esri.com/training/catalog/57630434851d31e02a43ef28/getting-started-with-gis/" rel="noopener noreferrer" target="_blank">Getting Started with GIS</a> (ArcMap)</li>
+      </ul>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="https://www.esri.com/training/catalog/57630435851d31e02a43f007/getting-started-with-arcgis-pro/" rel="noopener noreferrer" target="_blank">Getting Started with ArcGIS Pro</a> (Intro ArcGIS Pro)</li>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="https://www.esri.com/training/catalog/596e584bb826875993ba4ebf/cartography./" rel="noopener noreferrer" target="_blank">Cartography MOOC</a> (This is a great introduction to cartographic design principles and applications.)</li>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="https://www.esri.com/training/catalog/5b733e9d2fad23092c930883/arcgis-online-fundamentals/" rel="noopener noreferrer" target="_blank">AGOL Fundamentals</a> (This includes short intro videos; one intro course; and several advanced, maintenance required courses.)</li>
+    </ul>
+    <h3 class="text-xl"><a href="https://www.esri.com/training/" rel="noopener noreferrer" target="_blank">AGOL Basics and Essentials</a></h3>
```

Comment:
**Suggestion:** Remove link, which is redundant with the link in row 32. Anna is also removing this link from the pdf.

Reply:
tip: you could have edited this comment.